blog/source/_posts/WhyFbg.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Why FreeBoardGame.org?
+date: 2019-05-28 21:32:00
+tags: why motivation boardgame.io freeboardgame.org
+---
+
+One of the main pain points I experienced trying to play board games online with friends are all the fragmented implementations out there. Some amazing games only work for iOS, or others only for Android, or some only in Desktop. Most of these versions don't have multiplayer modes that can be played with each other, some are extremely expensive, and most require their own account. Besides, having one friend with the "wrong" OS will make it impossible for the party of friends to play together.
+
+So if there are so many board game implementations out there, how would someone in their sane mind think that [yet another board game project would help ?](https://xkcd.com/927/) 
+
+![XKCD](https://imgs.xkcd.com/comics/standards.png)
+
+Our bet is that FreeBoardGame.org is different: While most implementations out there are closed source, we are a Free software project. That means that instead of competing with each other in creating yet another game, we are going to slowly but surely share our work and create games that can be played, modified, improved, studied or even forked by anyone.
```

Comment:
"Free" in "Free software project" should be lowercase

Reply:
You are technically correct, but this is for emphasis